android-tests/build.gradle

```diff
@@ -0,0 +1,61 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        // Android Gradle plugin
+        classpath 'com.android.tools.build:gradle:1.3.1'
+
+        // Resolves android-sdk dependencies
+        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
```

Comment:
Well, would be nice to see new releases! Sometimes it can not handle
support libraries updates. But anyway, great tool.

On Fri, Nov 13, 2015, 07:52 Jake Wharton <notifications@github.com> wrote:

> In android-tests/build.gradle
> <https://github.com/ReactiveX/RxJava/pull/3519#discussion_r44749727>:
>
> > @@ -0,0 +1,61 @@
> > +buildscript {
> > +    repositories {
> > +        jcenter()
> > +    }
> > +
> > +    dependencies {
> > +        // Android Gradle plugin
> > +        classpath 'com.android.tools.build:gradle:1.3.1'
> > +
> > +        // Resolves android-sdk dependencies
> > +        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
>
> I'm amazed it even still works.
> … <#msg-f:1517698800156585010_>
> On Thu, Nov 12, 2015 at 11:51 PM Artem Zinnatullin <
> notifications@github.com> wrote: In android-tests/build.gradle <#3519
> (comment)
> <https://github.com/ReactiveX/RxJava/pull/3519#discussion_r44749702>>: >
> @@ -0,0 +1,61 @@ > +buildscript { > + repositories { > + jcenter() > + } >
> + > + dependencies { > + // Android Gradle plugin > + classpath
> 'com.android.tools.build:gradle:1.3.1' > + > + // Resolves android-sdk
> dependencies > + classpath
> 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0' It's for developers, I
> guess @akarnokd <https://github.com/akarnokd> <https://github.com/akarnokd>
> used it without noticing :) On Fri, Nov 13, 2015, 07:47 Jake Wharton <
> notifications@github.com> wrote: In android-tests/build.gradle <#3519
> <https://github.com/ReactiveX/RxJava/pull/3519> (comment) <#3519 (comment)
> <https://github.com/ReactiveX/RxJava/pull/3519#discussion_r44749555>>>: >
> @@ -0,0 +1,61 @@ > +buildscript { > + repositories { > + jcenter() > + } >
> + > + dependencies { > + // Android Gradle plugin > + classpath
> 'com.android.tools.build:gradle:1.3.1' > + > + // Resolves android-sdk
> dependencies > + classpath
> 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0' Not needed. .travis.yml
> takes care of this. — Reply to this email directly or view it on GitHub <
> https://github.com/ReactiveX/RxJava/pull/3519/files#r44749555>. --
> @artem_zin — Reply to this email directly or view it on GitHub <
> https://github.com/ReactiveX/RxJava/pull/3519/files#r44749702>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/3519/files#r44749727>.
>
-- 

@artem_zin